main.go

```diff
@@ -158,6 +161,9 @@ func parseArgs(a *app) func() int {
 		return func() int { return 1 }
 	}
 
+	if versionCmd.Happened() {
+		fmt.Printf("pgCarpenter version %s (git: %s)\n", Version, GitCommit)
```

Comment:
those are (old) changes from PR2. Could you rebase against PS2?

Reply:
PTAL